juriscraper/opinions/united_states/state/ohio.py

```diff
@@ -8,119 +8,68 @@
-
-        # Court drop down...
-        self.find_element_by_xpath(
-            "//select[@id='MainContent_ddlCourt']"
-            "/option[@value='{court}']".format(court=self.court_index)
-        ).click()
 
-        # Year drop down...
-        yearDropDownId = "MainContent_ddlDecidedYear"
-        yearDropDownPath = (
-            "//select[@id='{id}']/option[@value='%s']" % self.year
-        )
-        yearDropdownPaths = [
-            yearDropDownPath.format(id=yearDropDownId),  # Legacy examples
-            yearDropDownPath.format(
-                id=f"{yearDropDownId}Min"
-            ),  # current (2017)
-        ]
-        self.find_element_by_xpath(" | ".join(yearDropdownPaths)).click()
+    def _set_parameters(self) -> None:
+        """Set the parameters for the search
 
-        # Hit submit
-        submitPath = "//input[@id='MainContent_btnSubmit']"
-        self.find_element_by_xpath(submitPath).click()
-
-        # Selenium doesn't give us the actual code, we have to hope.
-        self.status = 200
-
-        text = self._clean_text(self.webdriver.page_source)
-        html_tree = html.fromstring(text)
-        html_tree.rewrite_links(
-            fix_links_in_lxml_tree, base_href=self.request["url"]
-        )
-        return html_tree
-
-    def _get_case_names(self):
-        path = f"{self.base_path}/preceding::td[1]"
-        case_names = []
-        for e in self.html.xpath(path):
-            case_names.append(
-                tostring(e, method="text", encoding="unicode").strip()
-            )
-        return case_names
-
-    def _get_download_urls(self):
-        path = f"{self.base_path}/preceding::td[1]//a[1]/@href"
-        return list(self.html.xpath(path))
-
-    def _get_docket_numbers(self):
-        path = f"{self.base_path}//text()"
-        return list(self.html.xpath(path))
-
-    def _get_summaries(self):
-        path = f"{self.base_path}/following::td[1]//text()"
-        return list(self.html.xpath(path))
+        :return: None
+        """
 
-    def _get_case_dates(self):
-        path = f"{self.base_path}/following::td[4]//text()"
-        dates = []
-        for s in self.html.xpath(path):
-            dates.append(
-                datetime.strptime(clean_if_py3(s).strip(), "%m/%d/%Y").date()
+        self.parameters = {
+            "__VIEWSTATEENCRYPTED": "",
+            "ctl00$MainContent$ddlCourt": f"{self.court_index}",
+            "ctl00$MainContent$ddlDecidedYearMin": f"{self.year}",
+            "ctl00$MainContent$ddlDecidedYearMax": f"{self.year}",
+            "ctl00$MainContent$ddlCounty": "0",
+            "ctl00$MainContent$btnSubmit": "Submit",
+            "ctl00$MainContent$ddlRowsPerPage": "50",
+            "__EVENTVALIDATION": self.html.xpath(
+                "//input[@id='__EVENTVALIDATION']"
+            )[0].get("value"),
+            "__VIEWSTATE": self.html.xpath("//input[@id='__VIEWSTATE']")[
+                0
+            ].get("value"),
```

Comment:
I’m open here.  I think seeing all the changes and then making a decision.  Not everything is the same or is set at the same time.  But it’s all so pretty simple. SD slightly excluded

Reply:
If you could give it a try tweaking it, that'd be great.

Reply:
That's better, but why set the year on the class instance? Why not set it in the method?

Reply:
no reason at all ... I thought you were suggesting it above.

Reply:
No,I was saying that rather than doing `self.year` in `__init__`, we should do `year =` in the `_set_parameters` method, so the year variable is set closer to where it is used.